### DISCUSSION TITLE
Quickly typing, completing , or selecting a user handle

### DISCUSSION THREAD

**Discussion by ghost** [community/community#119595] [Accessibility]
### Select Topic Area

Question

### Body

Hello and thanks for bearing with me.

It used to be (and memory is one of the better preserved parts of my aging self) that when you started typing something resembling a user handle (i.e. starting with an `@`) into a comment on a discussion or issue, a list of matching handles popped up and at that point you could navigate in the list with the arrow keys and finally hit `Tab` or `Enter` and the last selected handle would complete in the text entry field. The list still pops up, in fact, but sadly no keyboard shortcuts seem to be available while in that state. All I can do is remember the handle I intend to reference and type it myself after popping down the list.

What's interesting is that the same feature percolated to other web interfaces (Discourse, among others), *and now they seem to have the same problem*. This makes me think it's some kind of common javascript toolkit. Unfortunately, my javascript knowledge is too weak to track this deeper :-(

The outcome I hope for from this discussion is that I learn what the JS toolkit involved is so I can report a bug to them.

Thanks,

-- 
Ian

**Comment by PauGon75** [comment]
,

**Comment by vism2889** [comment]
I can also no longer tab-complete a username prefaced with `@`, the drop down of usernames is present but no way to hit enter or tab or another shortcut.  You have to click the name or type it manually.  This has been driving me insane for the past few weeks.

**Comment by cehfisher** [comment]
Hello @nobrowser - thank you for providing this feedback! We will file this with our Accessibility team so that it can be reviewed and will follow-up here with any additional questions or updates. Thanks again!

**Reply by ericwbailey** [reply]
Hi @nobrowser. I'm part of the GitHub accessibility team, and wanted to thank you for reporting this issue. 

I wanted to ask if you are still experiencing this issue? Same question for you, @vism2889. 

I am unable to reproduce the behavior on my test account. I evaluated using current Windows and macOS versions, using Blink, Webkit, and Gecko browsers without extensions installed.

**Reply by ghost** [reply]
@ericwbailey yes, it seems to work for me now. Thanks for caring.

-- 
Ian

**Comment by Jodes77** [comment]
> ### Select Topic Area
> 
> Question
> 
> ### Body
> 
> Hello and thanks for bearing with me.
> 
> 
> 
> It used to be (and memory is one of the better preserved parts of my aging self) that when you started typing something resembling a user handle (i.e. starting with an `@`) into a comment on a discussion or issue, a list of matching handles popped up and at that point you could navigate in the list with the arrow keys and finally hit `Tab` or `Enter` and the last selected handle would complete in the text entry field. The list still pops up, in fact, but sadly no keyboard shortcuts seem to be available while in that state. All I can do is remember the handle I intend to reference and type it myself after popping down the list.
> 
> 
> 
> What's interesting is that the same feature percolated to other web interfaces (Discourse, among others), *and now they seem to have the same problem*. This makes me think it's some kind of common javascript toolkit. Unfortunately, my javascript knowledge is too weak to track this deeper :-(
> 
> 
> 
> The outcome I hope for from this discussion is that I learn what the JS toolkit involved is so I can report a bug to them.
> 
> 
> 
> Thanks,
> 
> 
> 
> -- 
> 
> Ian
> 
>
